--- lxc.h.orig	 995500519 +0800
+++ lxc.h	 541639801 +0800
@@ -164,7 +164,7 @@
 /*
  * Returns the version number of the library
  */
-extern const char const *lxc_version(void);
+extern const char *lxc_version(void);

 #ifdef __cplusplus
 }
--- conf.h.orig	2012-03-12 17:57:57.995500519 +0800
+++ conf.h	2012-03-12 17:59:08.581687160 +0800
@@ -23,6 +23,10 @@
 #ifndef _conf_h
 #define _conf_h

+#ifdef __cplusplus
+extern "C" {
+#endif
+
 #include <netinet/in.h>
 #include <sys/param.h>
